bindings/nostr-ffi/src/event/tag.rs

```diff
@@ -483,6 +489,7 @@ pub enum TagEnum {
         public_key: Arc<PublicKey>,
         relay_url: Option<String>,
         alias: Option<String>,
+        uppercase: bool,
```

Comment:
Can you add the `/// Whether the p tag is an uppercase P or not` comment also here?